stable/prometheus/templates/statefulsets/server.yaml

```diff
@@ -25,12 +25,12 @@ spec:
     metadata:
     {{- if .Values.server.podAnnotations }}
       annotations:
-{{ toYaml .Values.server.podAnnotations | indent 8 }}
+        {{ toYaml .Values.server.podAnnotations | nindent 8 }}
     {{- end }}
       labels:
         {{- include "prometheus.server.labels" . | nindent 8 }}
-        {{- if .Values.server.statefulSet.labels}}
```

Comment:
Removed according to current [Deployment template](https://github.com/helm/charts/pull/23689/files#diff-32b95905f07752b90f3b19f28816ead2L32-L34).

`.Values.server.statefulSet.labels` is used to add labels to StatefulSet itself, not pods.